packages/backend/src/studio-api-impl.ts

```diff
@@ -367,7 +367,7 @@ export class StudioApiImpl implements StudioAPI {
     const unixPath: string = path.normalize(directory).replace(/[\\/]+/g, '/');
 
     podmanDesktopApi.env
-      .openExternal(podmanDesktopApi.Uri.file(unixPath).with({ scheme: 'vscode', authority: 'file' }))
+      .openExternal(podmanDesktopApi.Uri.file(unixPath).with({ scheme: 'vscode', authority: 'file/' }))
```

Comment:
@benoitf is this the expected behaviour ? I feel like this is an issue on the podman side ?

I am wondering, if it would be better to have instead something like above to match how URI are working

```
if(!unixPath.startsWith('/')) {
     unixPath = `/${unixPath}`;
}
```

![image](https://github.com/containers/podman-desktop-extension-ai-lab/assets/42176370/d6a12e54-88c1-40b0-9608-a7cec3d57e16)

Reply:
yes it feels invalid to have a / in the authority

first slash should be part of the unixPath if it requires one

also please add tests for openVSCode method for Windows and macOS